requirements.txt

```diff
@@ -22,7 +22,7 @@ pytest-watch==4.2.0
 python-dateutil==2.8.2
 python-rapidjson==1.8
 redis==4.3.4
-sentry-arroyo==2.17.1
+sentry-arroyo==0.1.68
```

Comment:
Why is this going back from 2.17.1 to 0.1.68? That does not look right

Reply:
oops, wrong lib. meant to open this on sentry-kafka-schemas